src/SablierComptroller.sol

```diff
@@ -287,19 +287,8 @@ contract SablierComptroller is ISablierComptroller, RoleAdminable {
         // Interactions: query the oracle decimals.
         uint8 oracleDecimals = AggregatorV3Interface(oracle).decimals();
 
-        // Adjust the price so that it has 8 decimals.
-        uint256 price8D;
-        if (oracleDecimals == 8) {
-            price8D = uint256(price);
-        } else if (oracleDecimals < 8) {
-            // The price is assumed to be much less than the maximum value of `uint256` so it is safe to multiply.
-            price8D = uint256(price) * 10 ** (8 - oracleDecimals);
-        } else {
-            price8D = uint256(price) / 10 ** (oracleDecimals - 8);
-        }
-
-        // Multiply by 10^18 because the native token is assumed to have 18 decimals.
-        return minFeeUSD * 1e18 / price8D;
+        // Calculate and return the value using the oracle decimals to avoid precision loss.
+        return minFeeUSD * 10 ** (10 + oracleDecimals) / uint256(price);
```

Comment:
I say lets refactor it as the following:

```solidity
if (oracleDecimals == 8) {
  return minFeeUSD * 1e18 / uint256(price);
} else {
  return minFeeUSD * 10 ** (10 + oracleDecimals) / uint256(price);
}
```

In 100% of the cases, the oracle decimal is going to be 8, so lets skip the extra maths there. The `else` provides safety net in case oracle decimal becomes $\neq$ 8.

Reply:
> not sure what you mean, but i agree with the suggestion

I meant that even though none of the USD pair returns decimals other than 8, we have the `else` condition as a safety feature just in case Chainlink changes this behaviour and starts returning non-8 decimals.